core/sources/domains.move

```diff
@@ -600,7 +641,7 @@ module aptos_names::domains {
 
     /// This is a shared entry point for clearing the address of a domain or subdomain
     /// It enforces owner permissions
-    fun clear_name_address(
+    public fun clear_name_address(
```

Comment:
Why do we want to make this public? We already have clear_domain_address and clear_subdomain_address which honestly are much easier and clearer to use (register_name should have had 2 separate functions for domain and subdomain as well instead of muddling them together, which makes it nearly impossible to read)